src/Components/IconEditorTab/IconEditorTab.scss

```diff
@@ -136,27 +137,72 @@
   .neoletter-form-icon-editor-tab .icon-editor-preview .bs-icon {
     color: #eee;
   }
+
+  .neoletter-form-icon-editor-tab .icon-editor-preview .button-del {
+    box-shadow: 0 0 4px 1px rgba(0, 0, 0, 0.8);
+  }
+
   #neoletter-icon-search .bs-icon {
     color: #aaa;
   }
   #neoletter-icon-search #search-input {
```

Comment:
Could you remove `tabIndex={1}` from `<input id="search-input"`? At least my eslint [is warning about it](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/65c9338c62c558d3c1c2dbf5ecc55cf04dbfe80c/docs/rules/tabindex-no-positive.md).